docs/source/table.rst

```diff
@@ -191,7 +191,15 @@
        - :ref:`genre`
      - 9579
      - .. image:: https://licensebuttons.net/l/by-nc-sa/3.0/80x15.png
-          :target: https://creativecommons.org/licenses/by-sa/3.0
+          :target: https://creativecommons.org/licenses/by-nc-sa/3.0
+
+   * - MTG_jamendo_autotagging_moodtheme
```

Comment:
do we need the `_` here (is it the official name)?

Reply:
done